src/commands/blockchain/transactionsByHeight.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ *
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {BlockHttp, QueryParams} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'h',
+        description: 'Block height',
+    })
+    height: number;
+
+    @option({
+        flag: 's',
+        description: 'Page size between 10 and 100, otherwise 10',
+    })
+    pageSize: number;
```

Comment:
It will be great to add the query params ``id`` and ``order`` as well.

Thanks!

Reply:
https://nemtech.github.io/nem2-sdk-typescript-javascript/classes/_infrastructure_queryparams_.queryparams.html

Reply:
Hi, What is the role of the query params id?

Reply:
Hello,

`order` lets you set the **direction of sorting**.

`id` lets you specify the ID of the last transaction you have read.

```
/**
                 * Page size between 10 and 100, otherwise 10
                 */
                public readonly pageSize: number,
                /**
                 * Id after which we want objects to be returned
                 */
                public readonly id?: string,
                /**
                 * Order of transactions.
                 * DESC. Newer to older.
                 * ASC. Older to newer.
                 */
                public readonly order: Order = Order.DESC,
```

Reply:
Okay, thank you!

Reply:
I looked at the command line method of nem2-cli, which seems to lack the way to display the transaction ` id`.

Reply:
transaction `id` is available in transaction meta: `Transaction.transactionInfo.id`

Reply:
Hi! The `id` is in meta, but it seems that the `id` of the display transaction is missing in `nem2-cli`. Is my operation wrong?